contracts/Tracer.sol

```diff
@@ -1,22 +1,23 @@
 // SPDX-License-Identifier: GPL-3.0-or-later
 pragma solidity ^0.6.12;
 
-import "@openzeppelin/contracts/math/SafeMath.sol";
-import "@openzeppelin/contracts/math/SignedSafeMath.sol";
-import "@openzeppelin/contracts/access/Ownable.sol";
+import "./lib/SafetyWithdraw.sol";
 import "./lib/LibMath.sol";
 import {Balances} from "./lib/LibBalances.sol";
 import {Types} from "./Interfaces/Types.sol";
-import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
-import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
 import "./Interfaces/IOracle.sol";
 import "./Interfaces/IInsurance.sol";
 import "./Interfaces/IAccount.sol";
 import "./Interfaces/ITracer.sol";
 import "./Interfaces/IPricing.sol";
 import "./DEX/SimpleDex.sol";
+import "@openzeppelin/contracts/math/SafeMath.sol";
```

Comment:
Just double checking, all these extra imports needed?

Reply:
Not extras I don't think?

Reply:
They're just shuffled around so yeah. Just reordering to make local contracts on top, and external ones beneath.